nengo/builder/connection.py

```diff
@@ -48,7 +60,7 @@ def build_linear_system(model, conn, rng):
     eval_points = get_eval_points(model, conn, rng)
     activities = get_activities(model, conn.pre_obj, eval_points)
     if np.count_nonzero(activities) == 0:
-        raise RuntimeError(
+        raise ZeroActivityError(
```

Comment:
Could we just make this a `BuildError` (and then get rid of `ZeroActivityError`)?  I think the only reason we had `ZeroActivtyError` before was so that we could provide a more specific error message for that particular case (without catching all `RunTimeErrors`, but now that's already happening here thanks to the refactoring.